### DISCUSSION TITLE
zpool iostat ZIL values

### DISCUSSION THREAD

**Discussion by zheliazkov** [openzfs/zfs#12316] [Q&A] [answered]
Hello,

I'm looking at the `zpool iostat -v` and I'm seeing that my log device doesn't get writes and reads and it is empty.

Do somebody know what can be the reason?

Thanks in advance.

BR,
Plamen

**Comment by thulle** [comment, marked as answer]
You're probably not doing any sync writes. Try setting sync=always on the filesystem being written to and you should get some statistics.

**Reply by zheliazkov** [reply]
That was the case. Thank you!
